### ISSUE TITLE
updating PIP?

### ISSUE THREAD

**Issue by tmz-pwn** [apogiatzis/gdb-peda-pwndbg-gef#12]
Updating pip (23.0.1 -> 24.0): pending.... 

I also had the issue where PIP update failed and a bunch of python errors popped up underneath..

**Comment by apogiatzis** [comment]
Good call! Thank you! I will update it when I get some free time. In the meantime, feel free to open a PR! :)

**Comment by tmz-pwn** [comment]
I think this is with pwndbg , unsure of others. Just running .\setup script for pwndbg by itself gives same issue

**Comment by hadrian3689** [comment]
> I think this is with pwndbg , unsure of others. Just running .\setup script for pwndbg by itself gives same issue

I had this issue recently because I haven't ran the script in a while. As of the time of writing it looks like pwndbg needs `python3.12` along with `python3.12-venv` installed.

Just running those too before running this script worked for me.

Usually pwndbg or any of the other tools will work with the most current Python version or the previous one so just a matter of initial diagnosis.

**Comment by John-Jkar** [comment]
I am trying to run the script but it telling me six moves module not found. I have tried installing it but its not working
